src/bindings/python/oglapphelpers/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+set(SOURCES
+        PyOpenColorIO_oglapphelpers.cpp
+        PyOpenColorIO_glsl.cpp
```

Comment:
Suggest naming the later two .cpp files just `Pyglsl.cpp` and `Pyoglapp.cpp`. Keeping PyOpenColorIO_oglapphelpers.cpp is ok since that is the entry point for the Python module of the same name.